src/olympia/addons/templates/addons/impala/button.html

```diff
@@ -48,7 +50,16 @@
       {{ _('End-User License Agreement') }}
     </a>
   {% endif %}
-
+  {% if waffle.switch('webext-permissions') %}
+    <br>
+    <a class="webext-permissions badge" href="{{ url('addons.privacy', addon.slug) }}">
```

Comment:
It will be the former until the latter is requested :)
c&p fail, fixed in https://github.com/mozilla/addons-server/pull/4670/commits/04851f1bdb8d98a691607a06c95ea75145efaac1